data/toc.yaml

```diff
@@ -2057,6 +2057,8 @@ Manuals:
       title: Continuous integration
     - path: /build-cloud/optimization/
       title: Optimization
+    - path: /build-cloud/private-resource-access/
```

Comment:
Should we remove this page from the PoC and keep it unlisted until we validate the PoC with a customer?

Reply:
Depends! If we publish it as experimental, we'd get more people using it. We could add a link to a Google form or a Qualtrics survey, where people can leave feedback, at the top of the page. WDYT?